### DISCUSSION TITLE
Using in NextJs 13 and having issues with internal import of css

### DISCUSSION THREAD

**Discussion by Tomas-Burian-TheMama** [epam/ketcher#2605] [Q&A]
When I try to use the module in NextJs 13 project, I'm constantly getting an error message:
./node_modules/draft-js/dist/Draft.css
Global CSS cannot be imported from within node_modules.

Is there a way to use Ketcher with NextJs? I couldn't find a single way to make it run, mainly because of this CSS issue.

**Comment by vishalmoltech** [comment]
I'm currently facing the same problem and, despite my efforts, I haven't been able to find a solution. It seems to be causing some roadblocks in my project development. If you have any insights or solutions on how to resolve this issue, I would greatly appreciate your assistance.
